pkg/systemd/overview.jsx

```diff
@@ -138,10 +138,10 @@ class OverviewPage extends React.Component {
                 {this.state.rebootModal && <ShutdownModal onClose={() => this.setState({ rebootModal: false })} />}
                 {this.state.shutdownModal && <ShutdownModal shutdown onClose={() => this.setState({ shutdownModal: false })} />}
                 <Page>
-                    <PageSection variant={PageSectionVariants.light} padding={{ default: 'noPadding' }}>
+                    <PageSection variant={PageSectionVariants.light} padding={{ default: 'noPadding' }} className="ct-pagesection-mobile">
                         <SuperuserAlert />
                     </PageSection>
-                    <PageSection className='ct-overview-header' variant={PageSectionVariants.light}>
+                    <PageSection variant={PageSectionVariants.light} className='ct-overview-header'>
```

Comment:
Same?

Reply:
Probably another one with some content inside that isn't holding a card view. (This time, the heading with system name and reboot.)